packages/angular/cli/src/commands/cache/info/cli.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { tags } from '@angular-devkit/core';
+import { promises as fs } from 'fs';
+import { join } from 'path';
+import { Argv } from 'yargs';
+import {
+  CommandModule,
+  CommandModuleImplementation,
+  CommandScope,
+} from '../../../command-builder/command-module';
+import { isCI } from '../../../utilities/environment-options';
+import { getCacheConfig } from '../utilities';
+
+export class CacheInfoCommandModule extends CommandModule implements CommandModuleImplementation {
+  command = 'info';
+  describe = 'Prints disk cache configuration and statistics in the console.';
+  longDescriptionPath?: string | undefined;
+  static override scope: CommandScope.In;
+
+  builder(localYargs: Argv): Argv {
+    return localYargs.strict();
+  }
+
+  async run(): Promise<void> {
+    const { path, environment, enabled } = getCacheConfig(this.context.workspace);
+
+    this.context.logger.info(tags.stripIndents`
+      Enabled: ${enabled ? 'yes' : 'no'}
+      Environment: ${environment}
+      Path: ${path}
+      Size on disk: ${await this.getSizeOfDirectory(path)}
+      Effective status on current machine: ${this.effectiveEnabledStatus()}
+    `);
+  }
+
+  private async getSizeOfDirectory(path: string): Promise<string> {
+    const directoriesStack = [path];
+    let size = 0;
+
+    while (directoriesStack.length) {
+      const dirPath = directoriesStack.pop();
+      if (dirPath === undefined) {
+        break;
+      }
+
+      let entries: string[] = [];
+      try {
+        entries = await fs.readdir(dirPath);
+      } catch {}
+
+      for (const entry of entries) {
+        const entryPath = join(dirPath, entry);
+        const stats = await fs.stat(entryPath);
+
+        if (stats.isDirectory()) {
+          directoriesStack.push(entryPath);
+        }
+
+        size += stats.size + stats.blksize;
```

Comment:
Isn't `blksize` the block size? Is `stats.size` enough? Otherwise, wouldn't it be `stats.blocks * stats.blksize`?

Reply:
Yeah size should be enough here.